sectoraccessor/sectoraccessor.go

```diff
@@ -0,0 +1,14 @@
+package sectoraccessor
```

Comment:
Do we _need_ the new package? Seems like all usages are from `retrievalmarket`, so this interface could just live there? (really trying to curtail the proliferation of micro-packages)

Reply:
I went back and forth on it - we also use this package from the miner API so it's not strictly to do with retrieval market. Fine with me to put it in the retrieval market package though.